packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -84,9 +84,17 @@ class SlidePanel extends React.Component {
       // Save the disclosing node for returning focus when panel is closed
       this.setDisclosingNode(this.lastClicked);
       this.panelNode.focus();
-    } else if (!this.props.isOpen && this.props.isOpen !== prevProps.isOpen) {
-      // Return focus to the disclosing element
-      this.disclosingNode.focus();
+      return;
+    }
+
+    if (!this.props.isOpen && this.props.isOpen !== prevProps.isOpen) {
+      if (this.disclosingNode !== undefined) {
```

Comment:
Should this also check for `null`? Or just a falsy check: `if (this.disclosingNode)`

Reply:
I can add a check for `null. Good catch.

Reply:
Updated here: 4e32267ead59031f52336a8cfda6ecd143bb7c19